exercises/clock/HINTS.md

```diff
@@ -12,5 +12,6 @@ The function `fromInteger`, from `Num`, must convert minutes
 to 24 hour clock time. It is not necessary to have a sensible
 implementation of `abs` or `signum`.
 
-Your can use the provided signatures if you are unsure about the types, but
-don't let them restrict your creativity.
+You will find a dummy data declaration and type signatures already in place,
```

Comment:
If this is *too* different from the original ("Don't let them restrict your creativity"), please help me to combine the two in a sensible manner.